packages/grafana-data/src/field/fieldOverrides.ts

```diff
@@ -459,3 +459,16 @@ export function applyRawFieldOverrides(data: DataFrame[]): DataFrame[] {
 
   return newData;
 }
+
+export const STANDARD_FIELD_OPTIONS = [
```

Comment:
nits: any reason for the upper case? Usually only do that for value constants.

Reply:
well, isn't this a value constant? :)

Reply:
@torkelo probably means immutable values like numbers and strings.

Reply:
Removed this as it was hacking around standard color field config propert